### ISSUE TITLE
Add YouTube description template page

### ISSUE THREAD

**Issue by felixbade** [DOTAalto/pms#21]
When uploading the entries to YouTube, it will easily save an hour (3mins/entry) if the descriptions are correctly formatted instead of manually parsed from different sources – because this is not a place to make typos.

Last year's title format:

`{title} by {author} – Graffathon 2023 – {compo}`

For example:

`Gloom by zatch, Miitoe, Thunschun, om – Graffathon 2023 – Beginner`

Please note the usage of endash (`–`) instead of a hyphen (`-`)

Last year's description format:

```
{rank_ordinal} place in {compo} compo with {points} points.

Download the original demo: https://files.scene.org/view/parties/2023/graffathon23/{compo_slug}/{filename}

More info about Graffathon: https://graffathon.fi
```

For example:

```
1st place in Beginner compo with 129 points.

Download the original demo: https://files.scene.org/view/parties/2023/graffathon23/beginner/gloom.zip

More info about Graffathon: https://graffathon.fi
```

I was originally thinking of a text file for these, but I think rich text + preview images would make it easier to manually match the texts to a video uploadable. The format of the page could just be a for loop of:
- thumbnail (`<img>`)
- rendered title text (`<h2>`)
- rendered description text (`<p>`)
- `<hr>`

**Comment by felixbade** [comment]
The formats can be hardcoded in Graffathon 2024.

**Comment by felixbade** [comment]
Fixed in 74857152
